tests/foreman/api/test_provisioningtemplate.py

```diff
@@ -751,3 +751,73 @@ def test_positive_verify_chronyd_timesource_kickstart_template(
             assert f'timezone --utc {timezone}' in render
         else:
             assert f'timezone --utc {timezone} --ntpservers {ntp_server_value}' in render
+
+    @pytest.mark.rhel_ver_match(r'^(?!.*fips).*$')
+    def test_create_host_with_invalid_template(
+        self,
+        target_sat,
+        default_architecture,
+        default_partitiontable,
+        default_org,
+        default_location,
+    ):
+        """Create a host with an invalid template
+
+        :id: 3df3c4a9-0feb-426e-a8d7-d74460bd6d9e
+
+        :steps:
+            1. Create a new provisioning template with a content that is not valid (raising an error)
+            2. Assign the invalid template to operating system
+            3. Create a host with the invalid template
+            4. Check if the host creation fails
+
+        :expectedresults:
+            1. The host creation fails
+        """
+        template_kind = target_sat.api.TemplateKind().search(query={"search": "name = provision"})[
+            0
+        ]
+        medium = target_sat.api.Media(organization=[default_org]).create()
+        os = target_sat.api.OperatingSystem(
+            name=('qe_' + gen_string('alpha')),
+            description=gen_string('alpha'),
+            minor=gen_string('numeric'),
+            major=gen_string('numeric', 2),
+            family='Debian',
```

Comment:
I think `Debian` won't work since we are checking `kickstart_default` here 
```suggestion
            family='Red Hat',
```